src/RWXLoader.js

```diff
@@ -209,10 +209,27 @@ function applyTextureToMat( threeMat, folder, textureName, textureExtension = '.
 	textureWrapping = RepeatWrapping ) {
 
 	let loader = new TextureLoader();
+	let texturePath = null;
+
+	if ( textureName.endsWith( '.png' ) ) {
+
+		threeMat.alphaTest = 1;
+		textureExtension = '';
+
+	} else if ( textureName.endsWith( '.jpg' ) ) {
+
+		textureExtension = '';
+
+	} else {
+
+		textureExtension = '.jpg';
+
+	}
```

Comment:
May I suggest to test if the texture as no extension at all and assume the extension to be '.jpg' if so?
The issue right now is: imagine `textureName` gets something else than a '.jpg' or a '.png' at the end of it (let's say it's '.bmp' or '.gif'): then this will still be set `textureExtension` to `.jpg` and `texturePath` will hold something like `folder/mytexture.bmp.gif`

You can use a small local regex for this.

Reply:
Textures other than JPEGs should now load.